### ISSUE TITLE
search for quotation marks in card text

### ISSUE THREAD

**Issue by infinitenutshell** [Null-Signal-Games/netrunnerdb#763]
is there a way to do it?

**Comment by plural** [comment]
at the moment, no.  The search is based off the "stripped_text" and "stripped_title" fields which have diacritics and punctionation (mostly) removed.  You can take a peek at those in https://github.com/NetrunnerDB/netrunner-cards-json.

Marking this as wontfix right now, but we may revisit it once a bunch of other stuff is done.
